collection-scripts/gather_nmstate_logs

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Do we fail must-gather if one of the script fails ? for tha we need to do `set -e`

Reply:
That's probably something @RickJWagner or @sferich888 can answer better

Reply:
Given that must-gather might run in clusters that are having problems, we should probably tend to be more forgiving of errors.  (And let things run as bullet-proof as possible, even if it means we give extra leniency to components that don't complete their jobs error-free.)  We'd prefer to get as much information as possible, even if it's some parts are unavailable.